pages/glossary/autoplay.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Autoplay
+key: Autoplay
+unambiguous: true
+objective: false

```

Comment:
should this be true?

```suggestion
objective: true

```

Reply:
I don't think it is really objective now.
"user starting it" or "interacting with it" is not really objective, I think. Even "play"/"playing" might be open to discussion.

Reply:
I copied it from the play button definition!